From: Chad Monroe <chad@monroe.io>
Date: Wed, 14 Jan 2026 17:08:15 -0800
Subject: pinctrl: airoha: fix I2C1 master mode pinmux configuration

The i2c_func_group for I2C1 incorrectly uses GPIO_2ND_I2C_MODE_MASK
(BIT(0)) instead of GPIO_I2C_MASTER_MODE_MODE (BIT(13)) when
configuring the pinmux register at offset 0x0214.

Fixes: 1c8ace2d0725 ("pinctrl: airoha: Add support for EN7581 SoC")
Signed-off-by: Chad Monroe <chad@monroe.io>
---
 drivers/pinctrl/mediatek/pinctrl-airoha.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/pinctrl/mediatek/pinctrl-airoha.c
+++ b/drivers/pinctrl/mediatek/pinctrl-airoha.c
@@ -1104,8 +1104,8 @@ static const struct airoha_pinctrl_func_
 		.regmap[0] = {
 			AIROHA_FUNC_MUX,
 			REG_GPIO_2ND_I2C_MODE,
-			GPIO_2ND_I2C_MODE_MASK,
-			GPIO_2ND_I2C_MODE_MASK
+			GPIO_I2C_MASTER_MODE_MODE,
+			GPIO_I2C_MASTER_MODE_MODE
 		},
 		.regmap_size = 1,
 	},
